### PR TITLE
Sync service

### PR DESCRIPTION
**Pull request by JobdePater**
Sync functions with scripts that are used in the BBWP Service. This can be used as startpoint

### CODE REVIEW

**Comment by JobdePater** [comment]
there are still errors in tests

File: R/bbwp_meas_scores.R [hunk 1]
```diff
@@ -218,16 +191,12 @@ bbwp_meas_score <- function(B_SOILTYPE_AGR, B_GWL_CLASS,  A_P_SG, B_SLOPE, B_LU_
                             D_OPI_NGW, D_OPI_NSW, D_OPI_PSW, D_OPI_NUE, D_OPI_WB,
                             measures, sector){
   
-  effect_psw = psw_psg_medium = psw_psg_high = effect_nsw = nsw_drains = nsw_gwl_low = nsw_gwl_high = psw_noslope = NULL
-  effect_ngw = ngw_grassland = psw_bulbs = D_MEAs_NGW = D_MEAS_NSW = D_MEAS_NUE = effect_nue = D_MEAS_WB = effect_Wb = diary = NULL
-  arable = tree_nursery = bulbs = clay = sand = peat = loess = D_MEAS_TOT = id = NULL
-  D_MEAS_PSW = D_MEAS_NGW = D_MEAS_PSW = effect_wb = NULL
```
Comment (gerardhros) [inline R/bbwp_meas_scores.R:194]: why removing the visual bindings; in the service they were not required, but in a package they are

File: R/bbwp_meas_scores.R [hunk 2]
```diff
@@ -27,9 +27,6 @@ bbwp_meas_rank <- function(B_SOILTYPE_AGR, B_GWL_CLASS,  A_P_SG, B_SLOPE, B_LU_B
                            D_OPI_NGW, D_OPI_NSW, D_OPI_PSW, D_OPI_NUE, D_OPI_WB,
                            available_measures, sector){
   
-  effect_psw = psw_psg_medium = psw_psg_high = effect_nsw = nsw_drains = nsw_gwl_low = nsw_gwl_high = psw_noslope = effect_ngw = NULL
-  ngw_grassland = psw_bulbs = D_MEAS_NGW = D_MEAS_NSW = D_MEAS_NUE = effect_nue = D_MEAS_WB = effect_wb = diary = arable = tree_nursery = bulbs = NULL
-  clay = sand= peat = loess = D_MEAS_TOT = effect_costs = id = D_MEAS_PSW = NULL
```
Comment (gerardhros) [inline R/bbwp_meas_scores.R:30]: same comment. needed to remove?

**Review by gerardhros — APPROVED** [review]

File: R/bbwp_meas_scores.R (hunk 1 above)
Comment (JobdePater) [reply R/bbwp_meas_scores.R:194]: some were still in the bbwp service, so I thought that all others were not needed anymore. Aded them again

**Review by gerardhros — COMMENTED** [review]
you have to update the check function.

the measure list might not have a column name identical to a column name in the internal dt (both have now id) and so the merge gives not a full join
the measure list has only 2 measures, and selecting the top-5 is then quite difficult.

i also encountered an issue when the D_MEAS_?? is zero. Please check

**Comment by gerardhros** [comment]
after analysis on 28-may i think this fork is substantially outdated.
all checks are still failing.
i did a quick check and copied the sync with BBWP service from this fork into version 0.3.2.
i have not seen the updated LSW yet (not pushed?), you might make a new branch for that from the main.

**Pull request closed without merging**